websites/management/commands/markdown_cleaning/utils.py

```diff
@@ -25,8 +26,31 @@ def remove_prefix(string: str, prefix: str):
 def get_rootrelative_url_from_content(content: WebsiteContent):
     dirpath = remove_prefix(content.dirpath, "content/")
     filename = "" if content.filename == "_index" else content.filename
-    pieces = ["/courses", content.website.name, dirpath, filename]
-    return "/".join(p for p in pieces if p)
+    pieces = [content.website.url_path, dirpath, filename]
+    return "/" + "/".join(p for p in pieces if p)
```

Comment:
For most courses, `website.name` and `website.url_path` are the same. That is true for legacy courses. For newer courses, these two values can be different. And it is more appropriate to use `url_path`. So I've updated the utility accordingly.